specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -958,7 +1062,11 @@
           "format": "date-time",
           "type": "string"
         }
-      }
+      },
+      "requried": [
+        "editTime",
```

Comment:
The required values shouldn't be changing for anything other than string or object type properties in 2018.